src/k2/dto/Persistence.h

```diff
@@ -77,5 +77,55 @@ struct PlogSealResponse {
     K2_DEF_FMT(PlogSealResponse, sealedOffset);
 };
 
+struct PlogStatusRequest {
+    String plogId;
+    K2_PAYLOAD_FIELDS(plogId);
+    K2_DEF_FMT(PlogStatusRequest, plogId);
+};
+
+struct PlogStatusResponse {
+    uint32_t currentOffset;
+    bool sealed;
+    K2_PAYLOAD_FIELDS(currentOffset, sealed);
+    K2_DEF_FMT(PlogStatusResponse, currentOffset, sealed);
+};
+
+struct PlogCreateError : public std::exception {

```

Comment:
same deal with these exceptions, move to the implementation, and consider just using Status directly as you'll have to map them anyway

Reply:
fixed